app/views/tracks/exercises/show/_concepts_section.html.haml

```diff
@@ -0,0 +1,23 @@
+- if exercise.concept_exercise?
+  %section.completed-info
+    %h3 You’ve learnt #{pluralize exercise.taught_concepts.size, 'concept'} by completing this exercise.
+    .concepts
+      - exercise.taught_concepts.each do |concept|
+        = render ViewComponents::Concept.new(concept, user_track, :small)
+
+    = render ViewComponents::ProminentLink.new("See how your concept map has changed", track_concepts_path(track))
+    .explanation
+      To master the concepts, you can try completing more exercises for them.
```

Comment:
I don't know if this is possible, but it would be great if this could be a link to a page listing the practice exercises that practice these concepts.

Reply:
The tooltip on the learnt concepts will have the practice exercises on them. I think that's the best we can do in the space we have (else we'd end up potentially listing 6x6 exercises here)

Reply:
👍